unity/PitayaExample/Assets/Pitaya/PitayaBinding.cs

```diff
@@ -140,6 +114,46 @@ private static string _AndroidBuildNumber()
             return (packageInfo.Get<int>("versionCode")).ToString();
         }
 
+        private static string Platform()
+        {
+            string platform;
+            switch (Application.platform)
+            {
+                case RuntimePlatform.Android:
+                    platform = "android";
+                    break;
+                case RuntimePlatform.LinuxEditor:
+                case RuntimePlatform.LinuxPlayer:
+                    platform = "linux";
+                    break;
+                case RuntimePlatform.WindowsEditor:
+                case RuntimePlatform.WindowsPlayer:
+                    platform = "windows";
+                    break;
+                case RuntimePlatform.IPhonePlayer:
+                    platform = "ios";
+                    break;
+                case RuntimePlatform.OSXEditor:
+                case RuntimePlatform.OSXPlayer:
+                    platform = "mac";
+                    break;
+                default:
+                    platform = Application.platform.ToString();
+                    break;
+            }
+
+            return platform;
+        }
```

Comment:
Minor: this method would be a bit shorter and cleaner if you replace the string platform variable and "breaks" for direct returns.

Reply:
Agreed! I wanted to change the minimum possible from original code, but that's a safe change.